fb_hddtemp/README.md

```diff
@@ -0,0 +1,39 @@
+fb_hddtemp Cookbook
+====================
+The `fb_hddtemp` cookbook installs and configures hddtemp, an hard drive
+temperature monitoring utility.
+
+Requirements
+------------
+
+Attributes
+----------
+* node['fb_hddtemp']['run_daemon']
+* node['fb_hddtemp']['disks']
+* node['fb_hddtemp']['disks_noprobe']
+* node['fb_hddtemp']['interface']
+* node['fb_hddtemp']['port']
+* node['fb_hddtemp']['database']
+* node['fb_hddtemp']['separator']
+* node['fb_hddtemp']['run_syslog']
+* node['fb_hddtemp']['options']
+
+Usage
+-----
+To install hddtemp include `fb_hddtemp`. It will autoprobe the available disks
+by default; override this behavior by setting a list of devices to probe in
+`node['fb_hddtemp']['disks']` and a list of devices to use without probing
+for a working sensors in `node['fb_hddtemp']['disks_noprobe']`. The database 
+used for identifying drives is located in `/etc/hddtemp.db` by default; this 
+can be changed with `node['fb_hddtemp']['database']`.
+
+The daemon is disabled by default; you can enable it by setting the attribute
+`node['fb_hddtemp']['run_daemon']` to true. If enabled, it will listen
```

Comment:
nevermind this is from the default/sysconfig file. I see that now... see above about moving these attrs